tests/FileDetector.php

```diff
@@ -209,43 +210,80 @@ public function TryDeduceEngine( array $Files, array $Matches ) : ?string
 			return 'Engine.SCI';
 		}
 
-		//If I have PCK files it might be Godot
+		//If I have a PCK file it might be Godot
 		if( $has( 'Evidence.PCK' ) )
 		{
+			//This is a really long and annoying check. Basically we have two things to look for:
+			//1. A single .pck file named exactly "data.pck", and NO other pck files
+			//2. For every executable, a correspondingly named pck file, and no other pck files
```

Comment:
The second rule would exclude Godot 3.0+ games using multiple PCKs (it's not a common use case but might become more common as more people make games with DLCs, etc.).

I'll clarify what I think detection rules should be in the PR description as it's more convenient than in a review comment.

Reply:
Yes, this is an unfortunate limitation at the moment. Our main goal for right now is to accept some false negatives if it can avoid a lot of false positives.

I'm merging this now as it should vastly improve on what's already active, but I'm happy to discuss further refinement.